test/integration/tests/checkout/dimensions.js

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+/* eslint max-lines: 0 */
+
+import { wrapPromise, uniqueID } from 'belter/src';
+import { FUNDING } from '@paypal/sdk-constants/src';
+
+import { generateOrderID, runOnClick,
+    createTestContainer, destroyTestContainer
+} from '../common';
+
+describe(`paypal checkout component happy path with dimensions`, () => {
+
+    beforeEach(() => {
+        createTestContainer();
+    });
+
+    afterEach(() => {
+        destroyTestContainer();
+    });
+
+    it('should render checkout, then complete the payment with dimensions', () => {
+        return wrapPromise(({ expect, error }) => {
+            return runOnClick(() => {
+                return window.paypal.Checkout({
```

Comment:
Could you use the `onRender` test hook to assert that xprops.dimensions is the expected value as the input width/height?

Reply:
@aditya27marri and I synced up offline. The plan is to use this approach for testing the popup size: https://github.com/paypal/paypal-checkout-components/pull/1741